code/modules/surgery/organs/organ_internal.dm

```diff
@@ -42,6 +45,8 @@
 
 /obj/item/organ/internal/remove(mob/living/carbon/M, special = 0)
 	owner = null
+
+	processing_objects |= src
```

Comment:
this is going to result in a few things like body eggs processing outside of the body, I'd imagine.